ruby-challenges.rb

```diff
@@ -11,6 +11,20 @@
 # Expected output: ['tea', 'water', 'soda water']
 
 
+def contains_letter (arr, letter)
+ result = []
+ arr.each do |value|
+   if value.include? letter
+     result << value
+   end
+ end
+ result
+end
+
+p contains_letter(beverages_array, letter_o)
+p contains_letter(beverages_array, letter_t)
+
```

Comment:
Looks great. I like the shovel operator. Another option could be using `.select` to capture the values in an array.